config/kdump.py

```diff
@@ -1,44 +1,69 @@
-import os
+import sys
+
 import click
 import utilities_common.cli as clicommon
-from swsscommon.swsscommon import ConfigDBConnector
+from swsssdk import ConfigDBConnector
+
 
 @click.group(cls=clicommon.AbbreviationGroup, name="kdump")
 def kdump():
-    """ Configure kdump """
-    if os.geteuid() != 0:
-        exit("Root privileges are required for this operation")
+    """Modify configuration of kdump"""
+    pass
+
 
 @kdump.command()
 def disable():
-    """Disable kdump operation"""
+    """Disable kdump feature"""
     config_db = ConfigDBConnector()
-    if config_db is not None:
+    if config_db:
         config_db.connect()
+        kdump_table = config_db.get_table("KDUMP")
+        if "config" not in kdump_table:
```

Comment:
>"config" not in kdump_table [](http://example.com/codeflow?start=11&length=27)

In what situation user will meet this error? This `config` command will exit with error code, what should user do to make it working?

Reply:
If the configuration of `KDUMP` is not present  in `CONIFG_DB` or some keys are missing , then this error will be shown to user. 

The user can try to run the command `sudo config reload -y` to refresh the `CONFIG_DB` to solve this issue.